downtime/models.py

```diff
@@ -14,4 +14,4 @@ class Period(models.Model):
     objects = PeriodManager()
 
     def __str__(self):
-        return "Scheduled downtime"
\ No newline at end of file
+        return "Scheduled Downtime: %s to %s" % (self.start_time, self.end_time)
```

Comment:
If the start or end time is not set, then it would display 'None' instead of a date-time.